delegate-app/Main.hs

```diff
@@ -358,7 +358,9 @@ mbQueueAuctionPhases tick delegateEvents toClientsChannel = do
       case mSecsLeft of
         Nothing -> pure ()
         Just s -> do
-          threadDelay (fromInteger s * 1_000_000)
+          -- Preventing infinite loop on stage corner
+          let secsToWait = if s == 0 then 1 else s
```

Comment:
Are we not using this? What is the infinite loop you want to prevent btw? Maybe we could expand the comment a bit to give more details

Reply:
> Are we not using this? 

Using what?

> What is the infinite loop you want to prevent btw?

Awaiting 0 seconds in case of stages intervals corner.

Reply:
-- In case of on stage intervals corner secsToWait stutter at 0.

I was probably wrong, it should not be infinite loop, it just generates giant log on awaiting 0 seconds for 100500 time.